drivers/clock_control/Kconfig.stm32

```diff
@@ -66,7 +66,8 @@ config CLOCK_STM32_MCO1_SRC_LSE
 	depends on SOC_SERIES_STM32F4X || \
 	           SOC_SERIES_STM32F7X || \
 	           SOC_SERIES_STM32L4X || \
-	           SOC_SERIES_STM32H7X
+			   SOC_SERIES_STM32H7X || \
+			   SOC_SERIES_STM32H5X
```

Comment:
Please fix indentation.
Apply everywhere

Reply:
I have fixed the indentation and applied suggestions of @faloj
Sorry for making a bit of mess in this PR